tests/src/Context/ContentTypeContext.php

```diff
@@ -81,7 +81,12 @@ public function aFieldWithTheFollowingSettingsIsAddedToTheType($arg1, TableNode
           $field_name = 'field_' . $value;
           $is_field_exiting = field_info_field($field_name);
           if ($is_field_exiting) {
```

Comment:
Why do not you use the bovigo/assert function? 
In this case, the code could be:
`assert($is_field_exiting, isNotNull(), t('The field "@name" already exists and cannot be used for test purpose.', array('@name' => format_username($setting_name))));`